content/en/code_analysis/_index.md

```diff
@@ -41,6 +41,8 @@ Static Analysis
 Software Composition Analysis 
 : Scans the open source libraries that are imported into your repositories for known vulnerabilities. 
 
+<div class="alert alert-info">Datadog Software Composition Analysis can find vulnerable libraries across the Software Development Lifecycle (SDLC). Code Analysis summarizes results found by directly scanning your repositories. To view all vulnerabilities found in repositories and at runtime consolidated together, head to <a href="/security/application_security/software_composition_analysis" target="_blank">Application Security</a> for more details.</div>
```

Comment:
```suggestion
<div class="alert alert-info">Datadog Software Composition Analysis can find vulnerable libraries across the software development lifecycle (SDLC). Code Analysis summarizes results found by directly scanning your repositories. To view all vulnerabilities found in repositories and at runtime consolidated together, see <a href="/security/application_security/software_composition_analysis" target="_blank">Application Security</a> for more details.</div>
```

- "software development lifecycle" should be lowercase
- "head to" is idiomatic